index.html

```diff
@@ -2290,6 +2290,277 @@ <h2>Definition of Roles</h2>
+          <tr>
+            <th class="role-related-head" scope="row">Related Concepts:</th>
+            <td class="role-related"><a href="https://www.w3.org/TR/2007/REC-xforms-20071029/#ui-common-elements-item">XForms item</a></td>
+          </tr>
+          <tr>
+            <th class="role-scope-head" scope="row">Required Context Role:</th>
+            <td class="role-scope"><rref>list</rref></td>
+          </tr>
+          <tr>
+            <th class="role-mustcontain-head" scope="row">Required Owned Elements:</th>
+            <td class="role-mustcontain"> </td>
+          </tr>
+          <tr>
+            <th class="role-required-properties-head">Required States and Properties:</th>
+            <td class="role-required-properties"> </td>
+          </tr>
+          <tr>
+            <th class="role-properties-head" scope="row">Supported States and Properties:</th>
+            <td class="role-properties">
+              <ul>
+                <li><pref>aria-level</pref></li>
+                <li><pref>aria-posinset</pref></li>
+                <li><pref>aria-setsize</pref></li>
+              </ul>
+            </td>
+          </tr>
+          <tr>
+            <th class="role-inherited-head" scope="row">Inherited States and Properties:</th>
+            <td class="role-inherited">Placeholder</td>
+          </tr>
+          <tr>
+            <th class="role-namefrom-head" scope="row">Name From:</th>
+            <td class="role-namefrom">author</td>
+          </tr>
+          <tr>
+            <th class="role-namerequired-head" scope="row">Accessible Name Required:</th>
+            <td class="role-namerequired"> </td>
+          </tr>
+          <tr>
+            <th class="role-namerequired-inherited-head" scope="row">Inherits Name Required:</th>
+            <td class="role-namerequired-inherited"> </td>
+          </tr>
+          <tr>
+            <th class="role-childpresentational-head" scope="row">Children Presentational:</th>
+            <td class="role-childpresentational"> </td>
+          </tr>
+          <tr>
+            <th class="role-presentational-inherited-head" scope="row">Inherits Presentational:</th>
+            <td class="role-presentational-inherited"> </td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+    <div class="role" id="descriptionlist">
+      <rdef>descriptionlist</rdef>
+      <div class="role-description">
+        <p>A <rref>section</rref> containing <rref>descriptionitem</rref> and <rref>descriptionterm</rref> elements.</p>
+
+        <p>Description lists contain children whose <a>role</a> is <rref>descriptionitem</rref> and <rref>descriptionterm</rref> to represent a list of terms each having one or more descriptions.</p>
+
+        <p>Author requirements for <a>elements</a> whose <a>role</a> is <code>descriptionlist</code>:</p>
+        <ul>
+          <li>MUST only use an element with role <rref>descriptionterm</rref> as the first child in the description list.</li>
```

Comment:
Re: HTML's allowance to contain `dt` and `dd` elements within a `div`, should this have an...
``` 
or an element whose <a>role</a> is <rref>group</rref> which in turn contains children whose <a>roles</a> are <rref>descriptionterm</rref> and <rref>descriptionitem</rref>, the first child of which MUST have a role of <rref>descriptionterm</rref>.
```

I could go either way on adding this or not, since authors could add a `div role=none` to achieve the same allowance of HTML as well.  Thoughts?

Reply:
I don't see where HTML5 allows `dt` or `dd` elements outside the `dl` element context:

* [HTML5 DT element](https://www.w3.org/TR/html5/grouping-content.html#the-dt-element)
* [HTML5 DD element](https://www.w3.org/TR/html5/grouping-content.html#the-dd-element)

Can you provide a reference?

Reply:
content model for `dl`
> Either: Zero or more groups each consisting of one or more dt elements followed by one or more dd elements, optionally intermixed with script-supporting elements.
> Or: One or more div elements, optionally intermixed with script-supporting elements.

e.g.
```
<dl>
 <div>
  <dt> Last modified time </dt>
  <dd> 2004-12-23T23:33Z </dd>
 </div>
 <div>
  <dt> Recommended update interval </dt>
  <dd> 60s </dd>
 </div>
 <div>
  <dt> Authors </dt>
  <dt> Editors </dt>
  <dd> Robert Rothman </dd>
  <dd> Daniel Jackson </dd>
 </div>
</dl>
```